src/utils/i18nString.ts

```diff
@@ -0,0 +1,27 @@
+import i18n from "i18n-js";
+import { Translations } from "../common/i18n/translations/type";
+
+export const i18nString = <
+  T extends keyof Translations,
+  U extends keyof Translations[T]
+>(
+  componentName: T,
+  componentString: U,
+  dynamicContent?: Record<string, string | number>
+): string => {
+  return i18n.t(`${componentName}.${componentString}`, dynamicContent);
+};
+
+export const i18nErrorString = <
```

Comment:
Why must have separate utility function for this?

Reply:
Have refactored this to be a single utility function!